files/preStudies/frameworks.tex

```diff
@@ -0,0 +1,80 @@
+\\
+\textit{Meteor} comes with its own view layer named Blaze. This can be substituted with React or AngularJS if either of them are preferred.
+
+\subsubsection{NativeScript}
+\textbf{Language:} JavaScript
+\\
+\textbf{Description:}
+\\
+\textit{NativeScript} uses a javascript toolchain to compile the javascript code into native API’s. This is a huge advantage because it makes the app work exactly like it was written natively without ever writing any native code or knowing about the native API’s. The markup is written in XML which is then styled with CSS.
+It’s developed by Telerik, a big software company where you can opt-in for support through a paid license. It’s supported by Google and therefore works well with \textit{Angular} and TypeScript. With TypeScript, the ''nice to have'' feature, type safety is fulfilled. \textit{Bluetooth Low Energy} capabilities can be achieved with the npm package \textit{nativescript-bluetooth}, created by npm user eddyverbruggen. \textit{NativeScript} supports iOS, Android, \textit{Universal Windows Platform} and if \textit{Angular} is used it also supports building web applications \citep{preStudy:frameworks:nativescript}.
+
+
+\subsubsection{ReactNative}
+\textbf{Language:} JavaScript, Objective-C and Java
+\\
+\textbf{Description:}
+\\
+\textit{ReactNative} is a framework by Facebook Inc. It extends the React framework for building native apps. \textit{ReactNative} requires the developers to know Objective-C for iOS development and Java for Android development. This does not fulfill our requirement that it has to be cross-platform without the developers needing to know about the distinct platform languages and API’s.
+
+
+\subsubsection{Xamarin}
+\textbf{Language:} C\#
+\\
+\textbf{Description:}
+\\ 
+\textit{Xamarin} was recently purchased by Microsoft. It's a cross-platform framework for developing app for iOS, Android, Windows Phone and Windows Universal Platform. There are two main ways of developing with \textit{Xamarin}. \textit{Xamarin forms} which is the method that has the hightest amount of code-sharing between platforms. Estimated aroud 80-95\% code sharing depending on how specific the UI has to be. 
```

Comment:
It's -> Is is
iOS, Android, Windows Phone and Windows Universal Platform in italics.